testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
+                    "ColorScheme.ColorSchEnable": {
+                        "schema": {
+                            "title": "Use Color schema",
+                            "description": "Enable/Disable the use of a color schema.",
+                            "enum": [
+                                0,
+                                1
+                            ],
+                            "enumLabels": [
+                                "Normal",
+                                "Scheme"
+                            ],
+                            "default": 1
+                        }
+                    },
+                    "ColorScheme.ColorSchIndex": {
+                        "schema": {
+                            "title": "Color schema",
+                            "description": "Specifies the color schema to use.",
+                            "enum": [
+                                0,
+                                1,
+                                2,
+                                3,
+                                4,
+                                5,
+                                6
+                            ],
+                            "enumLabels": [
+                                "Custom",
+                                "Invert Brightness",
+                                "Invert Colors",
+                                "Yellow on Black",
+                                "Blue Dye",
+                                "Black and White",
+                                "White and Black"
+                            ],
+                            "default": 1
+                        }
+                    },
+                    // [ColorScheme] Custom settings
+                    "ColorScheme.ColorSchBrightness": {
+                        "schema": {
+                            "title": "Color schema brightness",
+                            "description": "Specifies the brightness of the screen to use.",
+                            "type": "integer",
+                            "minimum": 40,
+                            "maximum": 160,
+                            "default": 100
+                        }
+                    },
+                    "ColorScheme.ColorSchContrastScalar": {
+                        "schema": {
+                            "title": "Color schema contrast",
+                            "description": "Specifies the contrast of the screen to use.",
+                            "type": "integer",
+                            "minimum": 10,
+                            "maximum": 190,
+                            "default": 100
+                        }
+                    },
+                    "ColorScheme.ColorSchType": {
+                        "schema": {
+                            "title": "Color effect",
+                            "description": "Selects a effect to be used in screen colors.",
+                            "enum": [
```

Comment:
See, this is what I mean, this is obviously an enumerated bitmask, all the values are powers of two.

Reply:
this has been removed as part of cleaning up based on SR test WARNINGS